src/main/java/com/google/jenkins/plugins/credentials/oauth/P12ServiceAccountConfig.java

```diff
@@ -46,32 +56,97 @@
   private static final String DEFAULT_P12_SECRET = "notasecret";
   private static final String DEFAULT_P12_ALIAS = "privatekey";
   private final String emailAddress;
-  private String p12KeyFile;
+  @CheckForNull
+  private String filename;
+  @CheckForNull
+  private SecretBytes secretP12Key;
+  @Deprecated   // for migration purpose
+  @CheckForNull
+  private transient String prevP12KeyFile;
```

Comment:
ditto

Reply:
Same as above I'll revert this name change and change the file item instead